tests/test_roundtrip.rs

```diff
@@ -36,9 +36,9 @@ fn progress() {
 ///
 /// Skipping is generally needed to avoid timeouts, because we
 /// are testing lots of files.
-fn should_skip(rng: &mut Rng) -> bool {
-    let float = rng.next_f64();
-    float < CHANCES_TO_SKIP
+fn should_skip(rng: &mut RngCore) -> bool {
+    let float = rng.next_u64();
```

Comment:
Same problem of `f64` :)

Since `CHANCES_TO_SKIP <= 1`, by using `next_u64`, you're effectively skipping only if your result is `0`, which happens roughly 1/2^64 times, i.e. never :)

Reply:
Done